index.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
       'v0.1.3': '0x177Bf15e7E703F4980b7eF75A58dc4198f0f1172'
     },
     StowToken: {
-      latest: '0x4cdfbdec0aa003116bf030f249a8a7285cd6a184',
+      latest: '0x3cf4ef1fd0d7ffa53420c5447e43b8e8c3d33166',
       'v0.1.0': '0x4cdfbdec0aa003116bf030f249a8a7285cd6a184'
```

Comment:
Should switch the version history to match as well

Reply:
updated